site/content/docs/5.3/utilities/borders.md

```diff
@@ -133,6 +144,10 @@ Or, choose from any of the `.border-opacity` utilities:
 
 Add classes to an element to easily round its corners.
 
+{{< design-callout-alert >}}
+These border variants with **a radius css class different than `rounded` and `rounded-pill`** should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
These border variants **other than `.rounded`, `.rounded-pill` and `.rounded-circle`** should not be used because they do not respect the Orange Design System specifications.
```

Note: `.rounded-circle` could be used to render stickers or other rounded elements present in the DSM.

Reply:
I like my version better. I can add a `.`before the class names. I would be:
> These border variants with **a radius css class different than `.rounded` and `.rounded-pill`** should not be used because they do not respect the Orange Design System specifications.

What do like better @louismaximepiton, @hannahiss, @MewenLeHo?

Reply:
Can we use `.rounded`? I think we can only use `.rounded-pill` and `.rounded-circle`?
On the syntax side, I'd prefer the dot syntax for classes for sure, once done, I think there's no reason to repeat ourselves around the classes. The simpler the better ?

Reply:
OK so I kept the original sentence, added the dot for the classes, changed "css" to "CSS". Regarding `.rounded`, it's never used and some rounded elements in the DSM don't use exactly the same value as the `.rounded` one; I'm thinking for instance about the countdown in https://system.design.orange.com/0c1af118d/p/50e873-e-shop/b/046123.
See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2064/commits/1a18802429789b0774dcf36fde0a64c7fa268783.